tests/unit/test_report_junit.py

```diff
@@ -68,7 +68,7 @@ def test_pass(self, report_fix):
         <testcase name="/pass"/>
     </testsuite>
 </testsuites>
-""")
+""")  # noqa: E501
```

Comment:
It complains when running on my laptop, because of line 67 which is over 90 characters long:

```
 pytest -m flake8 --flake8 tests/unit/test_report_junit.py 
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.9.13, pytest-7.1.2, pluggy-1.0.0
rootdir: /home/happz/git/tmt/tests/unit, configfile: pytest.ini
plugins: flake8-1.1.1
collected 6 items / 5 deselected / 1 selected                                                                                                                                                

tests/unit/test_report_junit.py F                                                                                                                                                      [100%]

========================================================================================== FAILURES ==========================================================================================
________________________________________________________________________________________ FLAKE8-check ________________________________________________________________________________________
/home/happz/git/tmt/tests/unit/test_report_junit.py:1:1: F401 'unittest.mock.patch' imported but unused
/home/happz/git/tmt/tests/unit/test_report_junit.py:67:80: E501 line too long (95 > 79 characters)
/home/happz/git/tmt/tests/unit/test_report_junit.py:86:80: E501 line too long (95 > 79 characters)
/home/happz/git/tmt/tests/unit/test_report_junit.py:107:80: E501 line too long (95 > 79 characters)
/home/happz/git/tmt/tests/unit/test_report_junit.py:128:80: E501 line too long (95 > 79 characters)
/home/happz/git/tmt/tests/unit/test_report_junit.py:149:80: E501 line too long (95 > 79 characters)
```

Reply:
Ah, sorry, did not realize I already included a `.flake8` config with the `99` chars max. Will add it to the PR.

Reply:
Ok, if you want to incorporate such a change into this patch, there are a couple more `E501` to remove then.

Reply:
Yes, I'm already working on that.